pom.xml

```diff
@@ -29,11 +29,13 @@
 	</parent>
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<elasticsearch.version>6.5.3</elasticsearch.version>
+		<elasticsearch.version>6.5.1</elasticsearch.version>
```

Comment:
Why going back ?

Reply:
It was a mistake. When I did the fork i was working in version 6.5.1